### ISSUE TITLE
[Feature Request]: FDR Correction

### ISSUE THREAD

**Issue by darianyao** [jasp-stats/jasp-issues#2843]
### Description

HI. Thank you for your wonderful workong! I was wondering if there are any plans to implement FDR correction in the future? FDR correction is more commonly used and popular compared to FWER for handling multiple comparisons. It would be very useful for many users.

### Purpose

adding FDR correction to post hoc test 

### Use-case

Multiple comparision correction

### Is your feature request related to a problem?

many problems related to need to do multiple comparision correction

### Is your feature request related to a JASP module?

ANOVA

### Describe the solution you would like

adding FDR correction to post hoc test

**Comment by tomtomme** [comment]
@darianyao 
Thx for the request! Sounds like a good one. But it may take some time to implement since the queue of feature requests for the next version 0.20 is already quite full :D

**Comment by TarandeepKang** [comment]
Hi Team, On the subject of FDR (Benjamini & Hochberg, 1995) I would like to ask if you could also consider looking into this newer and improved version which I've only ever seen used in Graphpad prism.

Y. Benjamini, A. M. Krieger, and D. Yekutieli (2006). Adaptive linear step-up procedures that control the false discovery rate. Biometrika, 93(3):491–507.

https://www.graphpad.com/guides/prism/latest/statistics/stat_pros_and_cons_of_the_three_met.htm
it seems to be implemented in a few different R packages, from a quick search

Thanks and Best,

Tarandeep

**Comment by darianyao** [comment]
Thank you for your responses and consideration of this feature request! It would be great to see FDR correction as feature in JASP. 
I believe this functionality is needed by many users working with multiple comparisons. Thank you again for your time and efforts in improving JASP. :)

**Comment by JohnnyDoorn** [comment]
Yes, great idea! The ANOVA R code (which indeed uses emmeans for follow-up analyses) can be found in our jaspANOVA repository: https://github.com/jasp-stats/jaspAnova 
Have you found the other documentation already for working on modules?

**Comment by TarandeepKang** [comment]
Yup. Thomas and Jorris (is that his name?) pointed me to it.

**Comment by tomtomme** [comment]
Joris :D https://jasp-stats.org/team/joris-goosen/

**Comment by annamariamon** [comment]
Hi everybody! I’ve been using JASP for almost five years now, and it’s incredibly useful, since as a psychology major I don’t have a good handle on programming. In my lab they often use FDR abile doing analysis on Python, while until now  I was quite good with Tukey correction. How is the implementation of the FDR going? Thank you so much for you help!

**Comment by JohnnyDoorn** [comment]
@TarandeepKang is there any progress on this? I can also pick it up if you currently (or in the foreseeable future) don't have time for it, or can have a chat with you in case you got stuck somewhere.

**Comment by TarandeepKang** [comment]
My apologies @JohnnyDoorn  I was "signed off" by my doctor recently so haven't been able to make much progress I don't know when I will be fit again because we don't know exactly what my course of treatment is yet. So can I hand this to you and I will pick up another request whenever I'm back? Good news is I have managed to figure the internals of how you code, mostly. Things are stored in commonly used functions et cetera and then passed to different types of objects for possible display objects to be displayed in tables et cetera. Anyway, it looks like silly old me missed the feature deadline for the upcoming release?

When I do get back it would be possibly ready to have a chat about some issues I have been having especially with pushing and pulling with git :)

**Comment by TarandeepKang** [comment]
There I go with NHS (British health service) terminology, which I forget Dutch people probably don't understand! I simply mean that I have been diagnosed with an illness and given leave from work/my PhD. I don't yet know how long it will be. Not overly long, but I don't make promises I can't deliver either!

**Comment by TarandeepKang** [comment]
Hi @JohnnyDoorn I just wanted to leave you another message because it seems you not have seen my last? I'm still not back at work, not yet sure how long it will be until I am. I seem to have missed the future deadline for 0.20 or 0.95 or whatever you are calling it now! :-) Perhaps I can leave this request in your hands and we can talk about git can pick up something else when definitely back? 

With best wishes and apologies,

Tarandeep

**Comment by tomtomme** [comment]
@TarandeepKang 
and in additionwe have a 'Module development' channel on Mattermost. 
I think @boutinb can add you there. Please send him your mail-adress to b.boutin at uva.nl

**Comment by TarandeepKang** [comment]
Hi @JohnnyDoorn Just a quick follow-up to my email to your Amsterdam email address on 23 February, it would be great if we could arrange that meeting so we can get me better situated with the model structure and I can get this request finished! Of course I understand that you've been busy with the release of documents not to mention the commitments of your actual full-time job! I just didn't want anyone on the team to think that I have dropped the ball on this! :-)

All best as always,

Tarandeep

**Comment by TarandeepKang** [comment]
@darianyao That means the code I wrote for the FDR correction will be ready soon amongst some of the changes and fixes that Johnny has been working up! Although I'm not exactly sure when. Sorry it took me so long! I hope it will be useful. If you have discovered the module library, you just need to tick the beta option and update the ANOVA module, soon after this issue is closed, and it will be there.
